app/assets/javascripts/client/common_create/ToolDetailsCtrl.js

```diff
@@ -37,7 +37,11 @@
 
     vm.save = function(entity) {
       CreateService.loadDistrict(vm.district);
-      CreateService.saveAssessment(entity);
+      if(CreateService.context === 'inventory') {
```

Comment:
This is probably a good change to make.  It also jives with pushing the creation and API logic out of the controllers and into services.